renovate.json

```diff
@@ -1,9 +1,12 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:recommended"],
+  "extends": ["config:recommended", "schedule:weekly"],
   "dependencyDashboard": true,
-  "prConcurrentLimit": 5,
+  "prConcurrentLimit": 20,
   "prHourlyLimit": 200,
+  "lockFileMaintenance": {
+    "enabled": true
+  },
```

Comment:
question: Why do we want this? Recreating a lockfile from scratch is a good-but-messy way of recursively updating dependencies, but I would expect a tool like Renovatebot to be able to reason about updating transitive dependencies…unless this is an escape hatch for cases it cannot handle?

Relevant docs link: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance